### ISSUE TITLE
kde icon refresh 

### ISSUE THREAD

**Issue by cappelikan** [Zren/plasma-applets#131]
when the script is rebooted update kde, the Skype and other icons disappear and only the basic icons of kde are displayed, can this be fixed? my kde version 5.11.5

**Comment by Zren** [comment]
> when the script is rebooted update kde

Which widget? The Terminal Update widget?

**Comment by cappelikan** [comment]
i run script ./update from directory and kde restart with default icons in
panel others program icons dont show

2018-01-28 7:05 GMT+02:00 Chris Holland <notifications@github.com>:

**Comment by Zren** [comment]
Which widget?

**Comment by Zren** [comment]
Oh you are a contributor, so eventcalendar.

**Comment by Zren** [comment]
Have you tried restarting the computer?

**Comment by cappelikan** [comment]
yes after reboot with kde is ok , this bug manifested after run script
./update

2018-01-28 7:25 GMT+02:00 Chris Holland <notifications@github.com>:

**Comment by cappelikan** [comment]
1.png - screenshot before running ./update
2.png - screenshot after running ./update

2018-01-28 7:28 GMT+02:00 Sergiy Golovko <cappelikan@gmail.com>:

**Comment by Zren** [comment]
Images aren't visible. Reply on https://github.com/Zren/plasma-applets/issues/131
